models/src/models.js

```diff
@@ -1,11 +1,34 @@
 var mongoose = require("mongoose");
-var userSchema = new mongoose.Schema({
+var mongoose.Promise = global.Promise;
+var Schema = mongoose.Schema;
+var bcrypt = require('bcrypt-nodejs');
+
+
+var userSchema = new Schema({
     username: String,
     email: String,
     password: String,
     created_at: { type: Date, default: Date.now }
 });
 
+userSchema.pre('save', function(next){
+    console.log("sssf");
```

Comment:
NEVER EVER use meaningless console.log!!!
Event if you know it is temporary, always put meaningful message for it